lib/prawn-rails-forms/document_extensions.rb

```diff
@@ -12,6 +12,7 @@ def field_row(height:, units:)
     private
 
     def make_text_field(start, width, height, field:, value:, options: {})
+      options.merge! PrawnRailsForms.default_text_field_options
```

Comment:
I think this merge should be in the other direction. you want the `options` kwarg to override the defaults, right?

Reply:
If so, perhaps a test-case for that?

Reply:
https://github.com/umts/prawn-rails-forms/pull/10/commits/d975fe7b954fdc5ad80c132850621bc3366ad24c
<img width="1085" alt="screen shot 2017-12-18 at 9 47 18 am" src="https://user-images.githubusercontent.com/3988134/34111701-871f62a6-e3d8-11e7-8498-11d9ec8e7ce3.png">